services/QuillLMS/app/models/concerns/translatable_question.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module TranslatableQuestion
+  extend ActiveSupport::Concern
+  include HTTParty
+  include Translatable
+  INCORRECT_SEQUENCES = 'incorrectSequences'
+  FOCUS_POINTS = 'focusPoints'
+  CMS_RESPONSES = 'cms_responses'
+
+  def create_translation_mappings
+    super
+    [INCORRECT_SEQUENCES, FOCUS_POINTS, CMS_RESPONSES].each do |type|
+      create_data_translation_mappings(type:)
+    end
+  end
+
+  private def create_data_translation_mappings(type:)
+    data_for_translation_mappings[type].each do |field_name, translatable_text|
+      create_translation_mappings_with_text(field_name:, translatable_text:)
+    end
+  end
+
+  private def data_for_translation_mappings
+    {
+      CMS_RESPONSES => cms_responses,
+      INCORRECT_SEQUENCES => translatable_data(type: INCORRECT_SEQUENCES),
+      FOCUS_POINTS => translatable_data(type: FOCUS_POINTS),
+    }
+  end
+
+  private def cms_responses
+    response_body = JSON.parse(cms_data.body)
+
+    transform_translatable_array(response_body, CMS_RESPONSES, 'id')
+  end
+
+  private def cms_url = "#{ENV['CMS_URL']}/questions/#{uid}/responses"
+  private def cms_data
```

Comment:
This is purely stylistic, so don't feel like you have to do this, but I like one-liners even for memoized methods like this one.  That said, I know some people find the `= ... ||=` construction to be awkward, so leaving this as-is is fine.
```ruby
private def cms_data = @cms_data ||= HTTParty.get(cms_url)
```